src/ResourceManager/Dns/Commands.Dns/Records/NewAzureDnsRescordSet.cs

```diff
@@ -61,15 +61,17 @@ public class NewAzureDnsRecordSet : DnsBaseCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Do not fail if the record set already exists.")]
         public SwitchParameter Overwrite { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation.")]
```

Comment:
This is a breaking change.  The advice for this breaking change is to leave the parameeter add the ```[Obsolete(...)]``` attribute to the property, and not use it in cmdlet execution.  This gives users time to adapt to the change.